Expensify/expensify-common
# Make chokidar work inside the VM

@tgolen Please review. Same as the Web-E PR, this makes `watch` work inside the vm as well as outside.


### Fixed Issues
None, just making this tool more widely available.

# Tests
1. Delete node_modules folder.
2. Run `npm i && npm run grunt watch` inside the vm. Change a js file, see the watch work.
3. Delete node_modules folder
4. Run `npm i && npm run grunt watch` outside the vm. Change a js file, see the watch work.
